skopeo_cidev/Containerfile

```diff
@@ -3,9 +3,7 @@ ARG BASE_TAG=latest
 FROM ${BASE_NAME}:${BASE_TAG}
 
 # Required to keep perl & other tooling happy
-ENV LANGUAGE="en_US" \
-    LC_ALL="en_US" \
-    LANG="en_US.UTF-8"
+ENV LC_ALL="C"
```

Comment:
This might break with non-ASCII characters. Values to consider are `en_US.UTF-8` or perhaps `C.UTF-8`.

(I’m not following this closely, IIRC some of the locale data is not available in default installs, or default containers. I’d expect one of the two above (but _not necessarily `en_US`) to be available. Alternatively, the other locales might be available in packages named something like `glibc-langpack*`. Random googling finds https://github.com/containers/toolbox/issues/60 and https://fedoraproject.org/wiki/Changes/Glibc_locale_subpackaging .)

Reply:
Oh, okay.  Installing additional packages isn't a problem.  Whatever's needed so the tests run smoothly.

Reply:
If `C` works fine, it’s not worth the effort/time to change it yet again for theoretical purity.

I vaguely remember Python in particular being not very happy with `C` + `UTF-8`, but that’s probably not relevant to us.

Reply:
Ya, this image is dedicated for skopeo.  Anyway, yes I agree, if it's not a problem, no need to fix it.  Thanks for your help/insight.